### ISSUE TITLE
US_USELESS_SUPPRESSION_ON_* should report which annotation is useless

### ISSUE THREAD

**Issue by kohlschuetter** [spotbugs/spotbugs#3395]
Using spotbugs-maven-plugin 4.9.3.0 , I get warnings like:

```
[ERROR] Medium: Suppressing annotation on the method ... is unnecessary [...] At ...:[lines 74-77] US_USELESS_SUPPRESSION_ON_METHOD
```

However, if there is more than one suppressed annotation, it's a guessing game which annotation was actually useless.

Can the error include the useless annotation(s) please?

**Comment by gtoison** [comment]
This should be fixed by #3304

**Comment by JuditKnoll** [comment]
> This should be fixed by [#3304](https://github.com/spotbugs/spotbugs/pull/3304)

The link is to the wrong PR, the correct one is #3403
